VimeoUpload/Descriptor System/Descriptor.swift

```diff
@@ -35,7 +35,7 @@ public enum DescriptorState: String
     case Finished = "Finished"
 }
 
-public class Descriptor: NSObject, NSCoding
+open class Descriptor: NSObject, NSCoding
```

Comment:
I believe `ProgressDescriptor` subclasses `Descriptor`. I can make the class `Descriptor` open but keep its methods and properties `public` if you want.